Add numberOfActiveVersions property to RLMRealm (#6869)
realm/realm-cocoa: CHANGELOG.md

x.y.z Release notes (yyyy-MM-dd)
=============================================================
### Enhancements

> * Add ability to view the active number of a versions of a Realm by using `RLMRealm.numberOfActiveVersions`. The purpose of this is to

Reviewer: ```suggestion
* Add `RLMRealm.numberOfActiveVersions` which returns the number of versions of a Realm that's currently in use. The purpose of this is to
```
